test/test_autograd.py

```diff
@@ -6895,6 +6856,49 @@ def fn(v):
             nnz = 0 if empty_nnz else 5
             _test(sparse_size + dense_size, len(sparse_size), nnz, device)
 
+
+    @dtypes(*torch.testing.floating_and_complex_types())
+    def test_sparse_backward(self, device, dtype):
+        class FixedGradientFunction(Function):
```

Comment:
Would you comment on the design of this function? What is it intended to validate, and how does it use FixedGradientFunction?

Reply:
yeah and also I think we should use `gradcheck` in this test

Reply:
Note that this test is not new, is the same as the previous one. What I understand this test check `sparse_backward` for different cases (`dense + sparse` cases) and IMO it's clear using this way.